spec/features/help_modal_spec.rb

```diff
@@ -0,0 +1,51 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Help modal' do
+  let(:user) { create(:user) }
+
+  context 'when authenticated' do
+    before do
+      sign_in user
+      allow(Settings).to receive(:allow_sdr_content_changes).and_return(true)
```

Comment:
I believe this is the default value, so you can remove it unless we're varying it in this test.

Reply:
Removed